resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -245,4 +339,57 @@ function DoesRegistryKeyPropertyExist {
     $itemProperty = Get-ItemProperty -Path $Path  -Name $Name -ErrorAction SilentlyContinue
     return $null -ne $itemProperty
 }
+
+function Get-CombinedState {
+	param(
+		[Parameter(Mandatory)]
+		[BinarySettingState]$SmoothScrollListBoxes,
+		[BinarySettingState]$SlideOpenComboBoxes,
+		[BinarySettingState]$FadeOrSlideMenusIntoView,
+		[BinarySettingState]$ShowShadowsUnderMousePointer,
+		[BinarySettingState]$FadeOrSlideToolTipsIntoView,
+		[BinarySettingState]$FadeOutMenuItemsAfterClicking,
+		[BinarySettingState]$ShowShadowsUnderWindows,
+		[string] $AnimationEffectsProperty = 'UserPreferencesMask'

```

Comment:
This should be specified as a variable so that it can be reused rather than passed as a parameter: See example below:

https://github.com/microsoft/winget-dsc/blob/61d2ebf668659f0d3725e171d1c5db8ecbbe93be/resources/Microsoft.Windows.Developer/Microsoft.Windows.Developer.psm1#L614-L615